CMake Error at svnqt/CMakeLists.txt:139 (SET_TARGET_PROPERTIES):
	set_target_properties called with incorrect number of arguments.

--- origsrc/qsvn-0.8.3/src/svnqt/CMakeLists.txt	2013-11-14 20:34:07.978263500 -0600
+++ src/qsvn-0.8.3/src/svnqt/CMakeLists.txt	2013-11-14 20:37:09.556649200 -0600
@@ -138,7 +138,7 @@ ELSE(WIN32)
     SET(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib${LIB_SUFFIX})
     SET_TARGET_PROPERTIES(${svnqt-name}
         PROPERTIES
-        COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+        COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 ENDIF(WIN32)
 
 SET(LIB_MAJOR 6)
